src/Solver/Solver.php

```
@@ -118,15 +113,17 @@ protected function getAssets(Composer $composer, string $assetDir, array $packag
      * @param string $filename The filename of asset package.
      *
      * @psalm-return string[] The package name and the relative package path from the current directory
+     *
+     * @throws Exception
      */
-    protected function getMockPackagePath(PackageInterface $package, string $assetDir, string $filename): array
+    private function getMockPackagePath(PackageInterface $package, string $assetDir, string $filename): array
     {
         $packageName = AssetUtil::getName($package);
```

PHPMD (2.15.0)

Avoid using static access to class '\Foxy\Util\AssetUtil' in method 'getMockPackagePath'. (undefined)

(StaticAccess)